src/context.cc

```diff
@@ -2722,7 +2722,7 @@ TimeEntry *Context::Continue(
     }
 
     if (GUID.empty()) {
-        displayError("Missing GUID");
+        displayError(std::string(__FUNCTION__) + ": Missing GUID");
```

Comment:
It's good for debugging easier, but

I'm not sure that the present of `__FUNCTION__ + "Missing GUID"` to user interface is a good idea 🤔 since it's meaningfulness in term of ordinary user perspective 👀 

I suggest that we can remove it, and the fix `!guid.isEmpty()` is sufficient 👍

Reply:
`__FUNCTION__` is a macro that will convert to the wrapping method name: "Continue" in this case. User would see this string: "Continue: Missing GUID" in the error widget.

Reply:
The text "Missing GUID" itself is also not very easy to understand for the user. So I think it's ok to add this detail so users can give us more specific error message when they contact us with the issue.